### PR TITLE
test: Test app launch

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**
Depends on #1075

### CODE REVIEW

**Review by adrien-coye — COMMENTED** [review]
Looking good, added some tiny remarks. Thanks for testing this.

File: kDriveTests/kDrive/Launch/ITAppLaunchTest.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        accessToken: "",
+        expiresIn: 0,
+        refreshToken: "",
+        scope: "",
+        tokenType: "",
+        userId: 0,
+        expirationDate: Date()
+    ))
+
+    override func setUpWithError() throws {
```
[the remark this answers is not in the document]
Comment (PhilippeWeidmann) [reply kDriveTests/kDrive/Launch/ITAppLaunchTest.swift:39]: I agree, I think this aligned with your next comment, we should discuss what we want to mock

File: kDriveTests/kDrive/Launch/MockAccountManager.swift
[the remark this answers is not in the document]
Comment (adrien-coye) [reply kDriveTests/kDrive/Launch/MockAccountManager.swift:1]: discussed

File: kDriveTests/kDrive/Launch/ITAppLaunchTest.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrien-coye) [reply kDriveTests/kDrive/Launch/ITAppLaunchTest.swift:39]: discussed

**Review by adrien-coye — APPROVED** [review]
LGTM, I only had minor remarks.

**Review by adrien-coye — APPROVED** [review]
All good 👍

**Pull request merged**